flowzone.yml

```diff
@@ -177,6 +177,76 @@ on:
         type: string
         required: false
         default: ""
+      repo_config:
+        description: "Set to true to standardise repository settings after a successful run"
+        type: boolean
+        required: false
+        default: false
+      repo_allow_forking:
+        description: "Allow forking of an organization repository"
+        type: boolean
+        required: false
+        default: true
+      repo_default_branch:
+        description: "Set the default branch name for the repository"
+        type: string
+        required: false
+        default: main
+      repo_delete_branch_on_merge:
+        description: "Delete head branch when pull requests are merged"
+        type: boolean
+        required: false
+        default: true
+      repo_description:
+        description: "Description of the repository"
+        type: string
+        required: false
+        default: ${{ github.repository }}
+      repo_homepage:
+        description: "Repository home page URL"
+        type: string
+        required: false
+        default: ""
```

Comment:
Will this clobber the existing urls for repos that have them?

Reply:
It won't set an empty URL (default), but if the user sets a URL, it will set it and overwrite whatever is currently there.